aiogram/dispatcher/filters/builtin.py

```diff
@@ -12,6 +12,19 @@
 from aiogram.types import CallbackQuery, Message, InlineQuery, Poll, ChatType
 
 
+ChatIDArgumentType = typing.Union[typing.Iterable[typing.Union[int, str]], str, int]
```

Comment:
just a tiny point :)
we've already imported values for (Union, Iterable) types below (line 6)